index.html

```diff
@@ -0,0 +1,163 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link rel="stylesheet" href="about.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      rel="stylesheet"
+      href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet" />
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.15.4/css/all.css"
+    />
+
+    <title>Youth Conference</title>
+  </head>
+  <body>
+    <header>
+      <nav class="first">
+        <ul class="first-nav">
+          <li class="topnav"><i class="fab fa-facebook-f"></i></li>
+          <li class="topnav"><i class="fab fa-twitter"></i></li>
+          <li class="topnav">English</li>
+          <li class="topnav">My Page</li>
+          <li class="topnav">Logout</li>
+        </ul>
+      </nav>
+      <div class="nav2">
+        
+        <ul class="nav2-content">
+          <li class="nav2logo"></li>
+          <li class="nav2list"><a href="./index.html">Home</a></li>
+          <li class="nav2list"><a href="./about.html">About</a></li>
+          <li class="nav2list"><a href="#sponsors">Sponsors</a></li>
+          <li class="nav2list"><a href="#more-info">Contact us</a></li>
+        </ul>
+      </div>
```

Comment:
- [ ] For this project you should follow exactly the original design from Behance. As requested by the previous reviewer, Kindly make sure that your navigation design fit the original design from Behance. here is the original design:
![image](https://user-images.githubusercontent.com/71874425/182206347-d82af688-4dd9-49d7-a1a5-292a90aaca1e.png)